Library/Homebrew/tap_migrations.rb

```diff
@@ -15,7 +15,6 @@
   "bbcp" => "homebrew/head-only",
   "bcwipe" => "homebrew/boneyard",
   "bindfs" => "homebrew/fuse",
-  "blackbox" => "homebrew/boneyard",
```

Comment:
Please ensure you file a PR at the boneyard to remove this from there as well, Thanks!